index.js

```diff
@@ -9,9 +9,19 @@ async function action() {
   const token = core.getInput("token");
   const octokit = github.getOctokit(token);
 
+  const issueNumber =
+    github.context.issue?.number ?? core.getInput("issueNumber");
```

Comment:
Should the input override the environment? If you're explicitly passing the issue number, I think it makes sense to always use it

Reply:
Oh yeah 10000% agree

Fixed here: 2f109c089a0a48433dee65b6742b6e758cc83e81
Test added here: 0c7adcf360b3cb210fb2a08906db734e4d1bef39